Panic when unable to add-peer due to permissions

As an un elevated user:
```bash
~/wiretrustee ❯ wiretrustee add-peer --allowedIPs 10.30.30.2/32 --key MY3paM+bln5PEoNahHMjzTUg3embjlsVOEIhPqyFC1o=
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x15d5def]

goroutine 1 [running]:
github.com/wiretrustee/wiretrustee/cmd.glob..func1(0x1ba3fe0, 0xc0000bff40, 0x0, 0x4)
	/home/runner/work/wiretrustee/wiretrustee/cmd/addpeer.go:26 +0xcf
github.com/spf13/cobra.(*Command).execute(0x1ba3fe0, 0xc0000bfec0, 0x4, 0x4, 0x1ba3fe0, 0xc0000bfec0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1ba49e0, 0x0, 0x162fc20, 0xc00008c058)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/wiretrustee/wiretrustee/cmd.Execute(...)
	/home/runner/work/wiretrustee/wiretrustee/cmd/root.go:31
main.main()
	/home/runner/work/wiretrustee/wiretrustee/main.go:9 +0x2e
```

When performing the same action with `sudo`:
```bash
~/wiretrustee 15s ❯ sudo wiretrustee add-peer --allowedIPs 10.30.30.2/32 --key MY3paM+bln5PEoNahHMjzTUg3embjlsVOEIhPqyFC1o=
~/wiretrustee ❯
```

--- a/cmd/addpeer.go
+++ b/cmd/addpeer.go
@@ -22,13 +22,16 @@ var (
 				os.Exit(ExitSetupFailed)
 			}
 
-			config, _ := Read(configPath)
+			config, err := Read(configPath)
+			if err != nil {
+				log.Fatalf("Error reading config file, message: %v", err)
+			}
 			config.Peers = append(config.Peers, connection.Peer{
 				WgPubKey:     key,
 				WgAllowedIps: allowedIPs,
 			})
 
-			err := config.Write(configPath)
+			err = config.Write(configPath)
 			if err != nil {
 				log.Errorf("failed writing config to %s: %s", config, err.Error())
 				os.Exit(ExitSetupFailed)

--- a/cmd/up.go
+++ b/cmd/up.go
@@ -17,7 +17,10 @@ var (
 		Run: func(cmd *cobra.Command, args []string) {
 			InitLog(logLevel)
 
-			config, _ := Read(configPath)
+			config, err := Read(configPath)
+			if err != nil {
+				log.Fatalf("Error reading config file, message: %v", err)
+			}
 
 			myKey, err := wgtypes.ParseKey(config.PrivateKey)
 			if err != nil {

